monai/networks/nets/finetune_fc.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2020 - 2021 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Optional, Tuple, Union
+
+import torch
+
+from monai.networks.layers import Conv, get_pool_layer
+
+
+class FinetuneFC(torch.nn.Module):
+    """
+    Wrapper to customize the fully connected layer of a classification model or replace it by convolutional layer.
+    This module expects the output of `model layers[0: -2]` is a feature map with shape [B, C, spatial dims],
+    then replace the model's last two layers with a `pooling + conv`, or replace the last layer with a `linear`.
+
+    Args:
+        model: PyTorch model with fully connected layer at the end, support both 2D and 3D models.
+            typically, it can be a pretrained model in Torchvision, like:
+            ``resnet18``, ``resnet34m``, ``resnet50``, ``resnet101``, ``resnet152``, etc.
+            more details: https://pytorch.org/vision/stable/models.html.
+        n_classes: number of classes for the last classification layer. Default to 1.
+        dim: number of spatial dimensions, default to 2.
+        use_conv: whether use convolutional layer to replace the FC layer, default to False.
+        pool_size: if using convolutional layer to replace the FC layer, it defines the kernel size for `AvgPool`,
+            default to 7.
+        pool_stride: if using convolutional layer to replace the FC layer, it defines the stride for `AvgPool`,
+            default to 1.
+        bias: the bias value when replacing FC layer. if False, the layer will not learn an additive bias,
+            default to True.
+
+    """
+
+    def __init__(
+        self,
+        model: torch.nn.Module,
+        n_classes: int = 1,
+        dim: int = 2,
+        use_conv: bool = False,
+        pool_size: Optional[Union[int, Tuple[int, int]]] = 7,
+        pool_stride: Optional[Union[int, Tuple[int, int]]] = 1,
```

Comment:
this API should be more generic now
```suggestion
        pool = ("avg", {"kernel_size": 7, "stride": 1})
```

Reply:
sure, updated in the latest commit.
Thanks.